src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3458,4 +3498,39 @@ static public void setAllLocales(String code) {
     }
   }
 
+  @Override
+  public String created(String serviceName) {
+    return serviceName;
+  }
+
+  @Override
+  public String started(String serviceName) {
+    return serviceName;
+  }
+
+  @Override
+  public String stopped(String serviceName) {
+    return serviceName;
+  }
+
+  public static void setPeer(String fullKey, String actualName, String serviceType) {

```

Comment:
seems like this should be on the base service class, not runtime.

Reply:
No .. the opportunity to override meta information, cannot be done in the created service. It's "before" that lifecycle phase has begun.

Reply:
If it's "before" the lifecycle, it should be defined as part of the lifecycle,  no?  i mean. you're defining a step prior to the started phase of the lifecycle.

Reply:
Ok.. I think this is related to my other comment..  I think there shoudl be a step between created and started.. where you are configuring and initializing resources.